service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/WorkspaceCloneUtils.java

```diff
@@ -234,15 +236,16 @@ private static ReferencedResource buildDestinationGcsBucketReference(
             sourceBucketResource.getResourceLineage(),
             sourceBucketResource.getWorkspaceId(),
             sourceBucketResource.getResourceId());
+    WsmResourceFields.Builder<?> sourceCommonFieldsBuilder =
```

Comment:
sourceCommonFieldsBuilder -> destinationCommonFieldsBuilder

Same for rest of file, including in buildDestinationResourceCommonFields